Stratis.Bitcoin.Tests/BlockStore/LoopTests/BlockStoreLoopStepBaseTest.cs

```diff
@@ -15,6 +15,9 @@
 
 namespace Stratis.Bitcoin.Tests.BlockStore.LoopTests
 {
+    /// <summary>
+    /// Base test class for all the BlockStoreLoop tests
+    /// </summary>
     public class BlockStoreLoopStepBaseTest
```

Comment:
Also looking at the file, the whole class is missing XMLDOC, so it might be a good idea to add it. However, @dangershony might disagree with it.